docs/Labeled.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: "Labeled"
+---
+
+# `<Labeled>`
+
+`<Labeled>` adds a label on top of its child if the child exposes a `label` prop, or if it's a field with a `source` prop. It's used in Show layouts, to display both a field and its label. 
```

Comment:
`. It's used in Show layouts...`
This explanation could be improved (?), since it is used in other layouts as well

Reply:
Show layouts with an **s** ;-)